### DISCUSSION TITLE
Edit on GitHub button

### DISCUSSION THREAD

**Discussion by cooper645** [requarks/wiki#7239] [Help / Questions] [answered]
I am trying to add the same thing to my wiki. Your **"Edit on GitHub"** appears to be a button that floats between the Title Area and the top of the Wiki page.

Could you explain how you have created / added this please. I have searched this repo Q&A with no luck.

### Button in question
![Screenshot 2024-06-26 at 18 30 12](https://github.com/requarks/wiki/assets/76519278/68952a2d-a8eb-4250-88c7-5c533c5d77b4)

**Comment by esasse** [comment]
I'm trying to do the same thing. Searched for help here and found your question. Thanks for any help.

**Comment by NGPixel** [comment, marked as answer]
You need to enable it under Admin > General > Edit Shortcuts > Edit Menu Bar:

<img width="864" height="916" alt="image" src="https://github.com/user-attachments/assets/60fdf4ce-a690-4336-9485-e3f71f9c172a" />

**Reply by cooper645** [reply]
Amazing, thank you so much
